docs/SUMMARY.md

```diff
@@ -142,8 +142,8 @@
   * [.feastignore](reference/feature-repository/feast-ignore.md)
 * [Feature servers](reference/feature-servers/README.md)
   * [Python feature server](reference/feature-servers/python-feature-server.md)
-  * [\[Alpha\] Go feature server](reference/feature-servers/go-feature-server.md)
```

Comment:
why remove?

Reply:
This just a link the actual file remove in this PR #5063

Reply:
actually we should probably revert that PR as @shuchu has been doing work on the go feature server.

Reply:
If that's the plan, then I've added it back. But it will be broken until PR [#5063](https://github.com/feast-dev/feast/pull/5063) is reverted.

Reply:
@shuchu do you mind reverting?